recipes-core/images/core-image-minimal.bbappend

```diff
@@ -1,16 +1,11 @@
-IMAGE_FEATURES_append = " ssh-server-openssh"
+IMAGE_FEATURES:append = " ssh-server-openssh"
 
-IMAGE_INSTALL_append = " \
-   packagegroup-rte-core \
-   packagegroup-rte-utils \
-   packagegroup-rte-imx \
-   packagegroup-rte-stm \
-   packagegroup-rte-coreboot \
-   locale-base-en-us \
-   "
-
-# set root password
-inherit extrausers
-
-RTE_ROOT_PWD = "meta-rte"
-EXTRA_USERS_PARAMS = "usermod -P ${RTE_ROOT_PWD} root;"
+IMAGE_INSTALL:append = " \
+    packagegroup-rte-core \
+    packagegroup-rte-utils \
+    packagegroup-rte-imx \
+    packagegroup-rte-stm \
+    packagegroup-rte-coreboot \
+    locale-base-en-us \
+    "
+# set root passwordINHERIT += "extrausers"RTE_ROOT_PWD = "meta-rte"EXTRA_USERS_PARAMS = "usermod -P ${RTE_ROOT_PWD} root;"
```

Comment:
@tym2k1 This probably shouldn't be commented out.

Reply:
If that helps, in currently running RTEs the default pwd is `meta-rte` and it would be nice if that persists.

Reply:
@PLangowski @tym2k1 This is probably the last one to be resolved here